.palantir/revapi.yml

```diff
@@ -7,3 +7,9 @@ acceptedBreaks:
       new: "method org.gradle.api.provider.SetProperty<com.palantir.gradle.dist.ProductDependency>\
         \ com.palantir.gradle.dist.RecommendedProductDependenciesExtension::getRecommendedProductDependenciesProvider()"
       justification: "not really a break"
+  "4.26.0":
+    com.palantir.sls-packaging:gradle-recommended-product-dependencies:
+    - code: "java.class.defaultSerializationChanged"
```

Comment:
This should be fine as the java serialization is only used by Gradle, so all it will do invalidate caches.